src/workspaces/NewWorkspaceModal/NewWorkspaceModal.tsx

```diff
@@ -1,7 +1,7 @@
-import { Modal, TooltipTrigger } from '@terra-ui-packages/components';
+import { Icon, Modal, TooltipTrigger } from '@terra-ui-packages/components';
 import { delay } from '@terra-ui-packages/core-utils';
 import _ from 'lodash/fp';
-import { Fragment, ReactNode, useState } from 'react';
+import React, { Fragment, ReactNode, useState } from 'react';
 import { div, h, label, p } from 'react-hyperscript-helpers';
```

Comment:
Since this is a `tsx` file, I wouldn't expect it to contain any hyperscript helpers. Would you mind doing a follow-up PR after this one to fully convert this file?

Reply:
Sure